tests/test_askbot.py

```diff
@@ -384,6 +384,42 @@ def test_initialization(self):
         self.assertEqual(ab.tag, ASKBOT_URL)
 
     @httpretty.activate
+    def test_too_many_redirects(self):
+        """Test whether a too many redirects error is properly handled"""
+
+        question_api_1 = read_file('data/askbot/askbot_api_questions.json')
+        question_api_2 = read_file('data/askbot/askbot_api_questions_2.json')
+        question_html_2 = read_file('data/askbot/askbot_question_multipage_1.html')
+        question_html_2_2 = read_file('data/askbot/askbot_question_multipage_2.html')
+        comments = read_file('data/askbot/askbot_2481_multicomments.json')
+
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_QUESTIONS_API_URL,
+                               body=question_api_1, status=200)
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_QUESTIONS_API_URL,
+                               body=question_api_2, status=200)
+
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_QUESTION_2481_URL,
+                               location=ASKBOT_QUESTION_2481_URL, status=301)
+
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_QUESTION_2488_URL,
+                               body=question_html_2, status=200)
+
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_QUESTION_2488_URL,
+                               body=question_html_2_2, status=200)
+
+        httpretty.register_uri(httpretty.GET,
+                               ASKBOT_COMMENTS_API_URL,
+                               body=comments, status=200)
+
+        backend = Askbot(ASKBOT_URL)
+        _ = [question for question in backend.fetch(from_date=None)]
+
```

Comment:
To check whether this patch works, you need to check if the expected questions are returned. This list should contain all the questions but the one with id `2488`.